packages/cmf/src/cmfConnect.js

```diff
@@ -189,15 +189,15 @@ export default function cmfConnect({
 	mapStateToProps,
 	mapDispatchToProps,
 	mergeProps,
-	omitCMFProps = false, // will be removed and considered to true in 2.0.0
+	omitCMFProps,
 	withComponentRegistry = false,
 	withDispatch = false,
 	withDispatchActionCreator = false,
 	withComponentId = false,
 	...rest
 } = {}) {
 	const propsToOmit = [];
-	if (omitCMFProps) {
+	if (omitCMFProps === undefined || omitCMFProps === true) {
```

Comment:
You can replace this double check by just `if (omitCMFProps)` + default value `omitCMFProps = true`